<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_rename">
     <property name="text">
      <string>Rename Assets</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_import">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_export">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
